srcpkgs/qalculate/template

```diff
@@ -13,10 +13,9 @@ maintainer="Artem Zhurikhin <ashpool@xecut.net>"
 license="GPL-2.0-or-later"
 homepage="https://qalculate.github.io"
 distfiles="https://github.com/Qalculate/libqalculate/releases/download/v${version}/libqalculate-${version}.tar.gz"
-checksum=28c46dc8dd975f253627d80c55a6feab7c44d965dce3ceffefc3cafc9a97c457
+checksum=ef422aa54eac7c711ece65dd3a5cbc66370d3e17173465313201897c201e7d3e
 
 libqalculate-data_package() {
-	archs=noarch
```

Comment:
With the dropping of `noarch`, there is no need to have a separate `data` subpackage. Please drop this package entirely, moving its contents to the `libqalculate` subpackage.

Reply:
I'm not sure how to do this without hurting any existing installations. I added my attempt in a commit to this PR. In my case, the package upgrade went smoothly, but deleting the old `libqalculate-data` package resulted in deletion of the files which are now a part of `libqalculate`.

Reply:
Ok, I just found out about `replaces` and it worked. Commit updated.

Reply:
Remember to remove the symlink.